Headers/DebugServer2/Types.h

```diff
@@ -106,6 +106,11 @@ struct StopInfo {
     kReasonBreakpoint,
     kReasonSignalStop, // TODO better name
     kReasonTrap,
+#if defined(OS_WIN32)
+    kReasonMemoryError,
+    kReasonMathError,
+    kReasonInstructionError,
```

Comment:
They are, and that's what we do later on. But leaking SIG* numbers and
details of why we do this (because we are dealing with the GDB protocol) in
Windows classes is not a good idea IMO.

On Wed, Mar 30, 2016 at 12:32 PM, strager <notifications@github.com> wrote:

> In Headers/DebugServer2/Types.h
> <https://github.com/facebook/ds2/pull/240#discussion_r57949094>:
>
> > @@ -106,6 +106,11 @@ struct StopInfo {
> >      kReasonBreakpoint,
> >      kReasonSignalStop, // TODO better name
> >      kReasonTrap,
> > +#if defined(OS_WIN32)
> > +    kReasonMemoryError,
> > +    kReasonMathError,
> > +    kReasonInstructionError,
>
> Are these not translatable to SIGSEGV/SIGFPE/SIGILL signals?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/ds2/pull/240/files/da279dd327af5b2d20a9796c80817d66e80e2b71..dcee5fbd5253932e250a49640779f0f6c30928a2#r57949094>
>



-- 
Stephane Sezer